src/ERC721ContractMetadata.sol

```diff
@@ -176,4 +188,80 @@ contract ERC721ContractMetadata is
     function _baseURI() internal view virtual override returns (string memory) {
         return _tokenBaseURI;
     }
+
+    /**
+     * @notice Sets the address to receive royalties.
+     *
+     * @param newWallet The new wallet address.
+     */
+    function setRoyaltyAddress(address newWallet) external onlyOwner {
+        // Set the new royalty address.
+        _royaltyAddress = newWallet;
```

Comment:
might consider eg not letting this be set to 0-address

Reply:
added in https://github.com/ProjectOpenSea/seadrop/pull/21/commits/2c3670c9540e6158f3f722eb46d3519cc8b716d5